tests/unit/dogstatsd/test_statsd.py

```diff
@@ -545,7 +756,7 @@ def func(arg1, arg2, kwarg1=1, kwarg2=1):
         self.assertEqual('tests.unit.dogstatsd.test_statsd.func', name)
         self.assert_almost_equal(0.5, float(value), 0.1)
 
-    @pytest.mark.skipif(not is_higher_py35(), reason="Coroutines are supported on Python 3.5 or higher.")
+    @unittest.skipIf(not is_higher_py35(), reason="Coroutines are supported on Python 3.5 or higher.")
```

Comment:
We are using `pytest` to run unit tests in `tox.ini`. Both decorators should be supported but probably we should not change it to the same PR.
```suggestion
    @pytest.mark.skipif(not is_higher_py35(), reason="Coroutines are supported on Python 3.5 or higher.")
```

Reply:
I wasn't using tox in my local dev env and the base `unittest` library does not recognize `@pytest` so I kept getting test failures. Since `unittest` is the lowest common denominator in testing and the `@unittest.skipIf` works fine with both frameworks I figured that it made sense not to use a third-party-specific one to support more development envs. I can split this out into a separate PR for sure but I think that overall there's no need to use a bespoke decorator if the built-in `unittest` module one works (especially if it covers more use cases).